<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="holder">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>821</width>
      <height>951</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="input_widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>741</width>
      <height>941</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>501</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_input"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="load_file_btn">
          <property name="text">
           <string>Загрузить из файла</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="solve_btn">
          <property name="text">
           <string>Вычислить функцию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="res_widget" native="true">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>9</y>
      <width>731</width>
      <height>941</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>9</y>
       <width>681</width>
       <height>931</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="res_layout"/>
    </widget>
   </widget>
   <zorder>holder</zorder>
   <zorder>res_widget</zorder>
   <zorder>input_widget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
